src/main/java/io/jenkins/plugins/generate/GeneratePluginData.java

```diff
@@ -63,9 +63,33 @@ public void generate() {
         return plugin;
       })
       .collect(Collectors.toList());
+      plugins.forEach(p -> computeTrend(p, plugins));
     writePluginsToFile(plugins);
   }
 
+  private void computeTrend(Plugin plugin, List<Plugin> plugins) {
+    double dependentInstallPctLastMonth = plugins.stream().filter(p -> isDependency(p, plugin))
+      .map(p -> getInstallPct(p, 1))
+      .max(Double::compare).orElse(0.0);
+    double installPctLastMonth = getInstallPct(plugin, 1);
+    double installPctMonthBefore = getInstallPct(plugin, 2);
+    double independence = Math.max(0, 1 - dependentInstallPctLastMonth / installPctLastMonth);
+    int trend =  (int) ((installPctLastMonth - installPctMonthBefore) * independence * 1E4);
```

Comment:
whats this magic 1E4? is that just to make it a percentage type thing?

Reply:
I wanted to keep the number as an integer rather than a double. Scaling by factor of 1E4 gives values between 1 and 10,000 for all current plugins.

Reply:
worth extracting to a constant?